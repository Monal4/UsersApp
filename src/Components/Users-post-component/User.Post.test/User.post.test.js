import React from 'react';
import { shallow } from "enzyme";
import '../../../setup';
import UserPost from "../userPost.component";
import axios from 'axios'

describe('Users post component', () => {

    const TestPost = [
        {
          "userId": 1,
          "id": 1,
          "title": "sunt aut facere repellat provident occaecati excepturi optio reprehenderit",
          "body": "quia et suscipit\nsuscipit recusandae consequuntur expedita et cum\nreprehenderit molestiae ut ut quas totam\nnostrum rerum est autem sunt rem eveniet architecto"
        }];

    it('is a snapshot for userPost', () => {
        expect(shallow(<UserPost post={TestPost}/>)).toMatchSnapshot();
    });

    it('is a userPost api Test', () => {
        const response =  axios(`https://jsonplaceholder.typicode.com/posts?userId=1`);

        expect(response).not.toBeNull();
    });

});