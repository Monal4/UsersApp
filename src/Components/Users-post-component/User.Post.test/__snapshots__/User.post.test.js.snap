// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users post component is a snapshot for userPost 1`] = `ShallowWrapper {}`;
